README.md

```diff
@@ -42,13 +42,13 @@ GitHub Enterprise Server.
 - **[How does Backup Utilities differ from a High Availability replica?](docs/faq.md)**
 - **[Docker](docs/docker.md)**
 
-### Support
+### **Support**
 
-If you find a bug or would like to request a feature in Backup Utilities, please
-open an issue or pull request on this repository. If you have a question related
-to your specific GitHub Enterprise Server setup or would like assistance with
-backup site setup or recovery, please contact our [Enterprise support team][3]
-instead.
+If you find a bug or want to request the functionality of the backup utility,
+please open the issue at or submit a request for this repository. If you have a 
+question related to your specific GitHub Enterprise Server setup or would like 
+assistance with backup site setup or recovery, please contact our 
+[Enterprise support team][3] instead.
```

Comment:
From a support perspective I don't think this improves the quality of the readme. There are a few extra words added in here that make it more difficult to read than the current version.